src/plugins/MiradorImageTools.js

```diff
@@ -44,45 +33,28 @@ const Root = styled('div')(({ small, theme: { palette } }) => {
   const borderImageBottom = borderImageRight.replace('to bottom', 'to right');
   return {
     backgroundColor: alpha(backgroundColor, 0.8),
+    borderBottom: small ? border : 'none',
+    borderImageSource: small ? borderImageBottom : borderImageRight,
     borderRadius: 25,
+    display: 'flex',
+    flexDirection: 'row',
     position: 'absolute',
-    top: 8,
     right: 8,
+    top: 8,
     zIndex: 999,
-    display: 'flex',
-    flexDirection: 'row',
     ...(small && { flexDirection: 'column' }),
-    [ToggleContainer]: {
-      ...(small && {
-        borderBottom: border,
-        borderImageSource: borderImageBottom,
-        display: 'flex',
-      }),
-    },
-    [ToolContainer]: {
-      ...(!small && {
-        borderRight: border,
-        borderImageSource: borderImageRight,
-        flexDirection: 'row',
-      }),
-      ...(small && {
-        flexDirection: 'column',
-        borderBottom: border,
-        borderImageSource: borderImageBottom,
-      }),
-    },
```

Comment:
I think these styles supported some responsive design at very small screen sizes (to change the image tools vertical):

<img width="513" alt="Screenshot 2025-01-15 at 09 59 45" src="https://github.com/user-attachments/assets/1116c792-d8f1-4a75-b295-963329a91998" />

Without them, we're getting this blob arrangement:

<img width="466" alt="Screenshot 2025-01-15 at 10 00 06" src="https://github.com/user-attachments/assets/1d4314e7-5172-4084-a230-4ba42c7327ac" />

I think you removed them to fix this error:
> Uncaught Error: Component selectors can only be used in conjunction with @emotion/babel-plugin, the swc Emotion plugin, or another Emotion-aware compiler transform.

but I wonder if there's some other way to keep the responsive-ness?

Reply:
Thank you for catching that. I thought I preserved the styles but I will fix this!

Reply:
OK, I worked on this, it should be good now.

Reply:
Here are screenshots if helpful, this is the revised styling
<img width="608" alt="Screenshot 2025-01-15 at 8 12 20 PM" src="https://github.com/user-attachments/assets/14efe469-8bec-4c18-933f-95e4be7c221c" />


<img width="1512" alt="Screenshot 2025-01-15 at 8 11 55 PM" src="https://github.com/user-attachments/assets/d6657b58-f688-4918-a7d0-4eff91b3ec0b" />